app/cisc/lib.go

```diff
@@ -0,0 +1,172 @@
+package main
+
+import (
+	"crypto/rand"
+	"crypto/rsa"
+	"crypto/x509"
+	"encoding/pem"
+	"errors"
+	"io/ioutil"
+	"os"
+	"sort"
+	"strings"
+
+	"golang.org/x/crypto/ssh"
+
+	"github.com/dedis/cothority/app/lib/config"
+	"github.com/dedis/cothority/log"
+	"github.com/dedis/cothority/network"
+	"github.com/dedis/cothority/services/identity"
+	"gopkg.in/codegangsta/cli.v1"
+)
+
+// loadConfig will return nil if the config-file doesn't exist. It tries to
+// load the file given in configFile.
+func loadConfig() error {
+	log.Lvl2("Loading from", configFile)
+	buf, err := ioutil.ReadFile(configFile)
+	if err != nil {
+		if os.IsNotExist(err) {
+			return nil
+		}
+		return err
+	}
+	_, msg, err := network.UnmarshalRegistered(buf)
+	if err != nil {
+		return err
+	}
+	var ok bool
+	clientApp, ok = msg.(*identity.Identity)
```

Comment:
Did it anyway and don't really like it. But well, globals are bad, even though I would've let them in the current case.